Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -219,24 +211,18 @@ struct CombinatorialKalmanFilterResult {
 /// The void components are provided mainly for unit testing.
 template <typename propagator_t, typename updater_t = VoidKalmanUpdater,
           typename smoother_t = VoidKalmanSmoother,
-          typename source_link_selector_t = CKFSourceLinkSelector,
-          typename branch_stopper_t = VoidBranchStopper,
-          typename calibrator_t = VoidMeasurementCalibrator>
+          typename branch_stopper_t = VoidBranchStopper>
 class CombinatorialKalmanFilter {
  public:
-  /// Shorthand definition
-  using MeasurementSurfaces = std::multimap<const Layer*, const Surface*>;
-  /// The navigator type
-  using KalmanNavigator = typename propagator_t::Navigator;
-
```

Comment:
I have a comment somewhere on `MeasurementSurfaces`, the change to `KalmanNavigator` has nothing to do with the title of this PR.